library-model-loader/build.gradle

```diff
@@ -32,18 +32,5 @@ dependencies {
     compileOnly deps.apt.autoService
     annotationProcessor deps.apt.autoService
     compileOnly deps.build.nullaway
-    annotationProcessor deps.build.nullaway
     compileOnly deps.build.guava
 }
-
-import net.ltgt.gradle.errorprone.CheckSeverity
-
-tasks.withType(JavaCompile) {
-    // remove the if condition if you want to run NullAway on test code
-    if (!name.toLowerCase().contains("test")) {
-        options.errorprone {
-            check("NullAway", CheckSeverity.ERROR)
-            option("NullAway:AnnotatedPackages", "edu")
-        }
-    }
-}
```

Comment:
Are we entirely removing running NullAway on this target here? Or do we get this from some other more general task in the build? (I think we shouldn't be running for `edu.*` in general here, but we should still be running for `edu.ucr.cs.riple.*`, no?)

Reply:
Hi @lazaroclapp I replied to your comment around `OnField.duplicate()` previously. Please find it [here](https://github.com/nimakarimipour/NullAwayAnnotator/pull/41#discussion_r947204318).
Regarding the `NullAway` enrollment, yes we need to run it under package name `edu.ucr.cs.riple.*` and I think we should plan to do this for the whole repo at some point. That enrollment, which only covers `LibraryModelsLoader` was due to my assumption that in order for `LibraryModelsLoader` to be picked up by `NullAway`, it should have `NullAway` in its annotation processor path which is not true and It wasn’t part of any `NullAway` enrollment plan. 
That's why I removed it for now. Will enroll all modules in NullAway under the name `edu.ucr.cs.riple.*` in near future.

Reply:
Ok, this all makes sense! Sorry about the confusion with `OnField.duplicate()`, GitHub seems to somehow split the thread when it includes comments that are part of two reviews 😓